hack/update-staging-client-go.sh

```diff
@@ -18,5 +18,58 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+V=""
+while getopts ":vs" opt; do
+  case $opt in
+    v) # increase verbosity
+      V="-v"
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      ;;
+  esac
+done
+readonly V
+
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-"${KUBE_ROOT}"/staging/copy.sh
+cd ${KUBE_ROOT}
+
+# Create a temporary GOPATH for apimachinery and client-go, copy the current master into each and turn them
+# into a git repo. Then we can run copy.sh with this additional GOPATH. The Godeps.json will
+# have invalid git SHA1s, but it's good enough as a smoke test of copy.sh.
```

Comment:
Yeah, will fix that comment, something along these lines: ".... invalid git SHA1s, but it's good enough for running copy.sh. The actual k8s.io/apimachinery dependency will be updated later by the github bot to point to the real k8s.io/apimachinery github commit created by the bot just before."